x-pack/platform/plugins/shared/spaces/public/spaces_context/wrapper_internal.tsx

```diff
@@ -60,11 +60,17 @@ export const SpacesContextWrapperInternal = (
   );
 
   useEffect(() => {
+    let unmounted = false;
     getStartServices().then(([coreStart]) => {
       const { application, docLinks, notifications } = coreStart;
       const services = { application, docLinks, notifications };
-      setContext(createSpacesReactContext(services, spacesManager, spacesDataPromise));
+      if (!unmounted) {
+        setContext(createSpacesReactContext(services, spacesManager, spacesDataPromise));
+      }
     });
```

Comment:
__nit:__ early return would be slightly easier to follow
```ts
    getStartServices().then(([coreStart]) => {
      if (unmounted) {
        return;
      }

      const { application, docLinks, notifications } = coreStart;
      const services = { application, docLinks, notifications };
      setContext(createSpacesReactContext(services, spacesManager, spacesDataPromise));
    });
```

Reply:
Updated here [`eddd482` (#206696)](https://github.com/elastic/kibana/pull/206696/commits/eddd482853b4edf3a54ac5b06fefb02213e5868c)